src/entities/card/card.module.css

```diff
@@ -19,14 +19,14 @@
     margin-right: 0;
 }
 
-.card :global .details {
+.card .details {
```

Comment:
now, when you already use css modules you don't need nesting
so you can replace 
```css
.card .details {}
```
to
```css
.details {}
```

Reply:
Yeap. But its not a problem with shallow nesting. Im using it just for visual comfort. But anyway, thanks for the reminder)